src/components/Timer/Timer.js

```diff
@@ -0,0 +1,104 @@
+import { html } from '../../shared';
+import styles from './Timer.module.css';
+
+/**
+ * @param {{ startingMinutes: number, onFinish: () => void }} props
+ * @returns {HTMLElement}
+ */
+
+function Timer({ startingMinutes, onFinish }) {
+  let count = startingMinutes * 60;
+  let counting;
+
+  const counter = html`<span class="${styles.counter}">00:00</span>`;
+
+  const formatDuration = (time) => {
+    const timeObj = new Date(time * 1000);
+
+    const checkIfLessThan10 = (timeUnit) => {
+      return timeUnit < 10 ? `0${timeUnit}` : timeUnit;
+    };
```

Comment:
Jest na to wbudowana metoda [`String.prototype.padStart()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart):
```JS
checkIfLessThan10(n) === n.toString().padStart(2, '0')
```

Reply:
Co do yarn.lock, to porównwałam wersję, którą mamy aktualnie i tą, która była zmieiona, rzeczywiście tak kilka linii było zmienionych, więc po prostu wgrałam tą wersję którą mieliśmy wcześniej i wszystko powinno być ok.